test/Tests.Infrastructure/AzureStorageEmulatorRunning.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace Tests.Infrastructure
+{
+    [CLSCompliant(false)]
+    public class AzureStorageEmulatorRunning : FactAttribute
```

Comment:
I would suggest renaming it to AzureStorageEmulatorFact, it would match naming of other Facts and Theories.